app/routes/movies.$id.jsx

```diff
@@ -2,42 +2,65 @@ import { json } from "@remix-run/node";
 import styles from "../styles/movie-details.css?url";
 import { Link, isRouteErrorResponse, useRouteError, useLoaderData } from "@remix-run/react";
 
+const apiKey = process.env.OMDB_API_KEY;
+const baseUrl = "http://www.omdbapi.com/";
```

Comment:
better make them [uppercase](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#basic_const_usage)
but it only applies for constant values and env variables and base URL are these cases